CONTRIBUTING.md

```diff
@@ -123,6 +124,96 @@ from the main (upstream) repository:
     ```
 
 
+## <a name="commit"></a> Commit Message Guidelines
+
+We have very precise rules over how our git commit messages can be formatted.  This leads to **more
```

Comment:
Should we mention here, that we want to use squash merge PR? Contributors don't really have to write proper formatted commit messages since their messages will disappear either way.

Reply:
- I dont think so contributor have to know how we merge PRs.
 - also `conventional-changelog` is standard in lot of open-sources - they will learn something new
 - commit message is used as PR title so we do not have to rename PRs if contributor use good description and our format

Reply:
I don't think we will be able to do anything if contributors write not standardized commit messages. Blocking merge just because of commit messages which are discarded anyway isn't the best option.
However we can write it here - at least some folks might use standardized messages.

As for the last bullet point - this might by true, but mostly only for one commit PRs. For bigger PRs we would still have at least cpy&paste them manually. But it if saves work at least in some situations it sounds good.

Reply:
Understand you. But it is necessary to write it in this document? I think we can just have some internal agreement that it is non-critical issue and we can merge also PRs with non-standard commit message format.